### PR TITLE
#shy-lizard npm-modules

### PR DESCRIPTION
**Pull request by debrayc**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<head>
+  <title> This is only a Test </title>
```
Comment (breyana) [inline public/index.html:2]: This is the only page that comes up when I attempt to load the site

File: routes/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET home page. */
+router.get('/', function(req, res, next) {
+  res.render('index', { title: 'NPM Modules for JS' });
```
Comment (breyana) [inline routes/index.js:6]: I think this is rendering index.html instead of index.pug?

File: spec/camelcase.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports =function ( str ) {
```
Comment (breyana) [inline spec/camelcase.js:1]: add a space after the equals sign

File: spec/ceil.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+module.exports = function (num1){
+  return Math.ceil(num1)
```
Comment (breyana) [inline spec/ceil.js:2]: I'd try rewriting `.ceil` in code instead of using `.ceil`

File: spec/divide.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+module.exports = function(num1 , num2){
+ // 1.5){
```
Comment (breyana) [inline spec/divide.js:2]: Leave out comments in the final product

File: spec/min.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+module.exports = function min( inputArray ) {
+
+    return Math.min.apply(null, inputArray)
```
Comment (breyana) [inline spec/min.js:3]: Try to rewrite `.min` instead of using `.min`

**Review by breyana — CHANGES_REQUESTED** [review]
Overall, try to rewrite the methods when doing NPM modules instead of using the methods. Also, when making tests, try to have more than one test case. Try to break your code by doing edge cases.

**Comment by debrayc** [comment]
thank you!...
